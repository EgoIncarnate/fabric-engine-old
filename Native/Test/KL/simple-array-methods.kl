/*
 *  Copyright 2010-2012 Fabric Technologies Inc. All rights reserved.
 */

operator entry()
{
  var Integer a[];
  report a + "; size " + a.size;
}
