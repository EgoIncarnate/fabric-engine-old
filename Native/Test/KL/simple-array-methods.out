GlobalList
  Function( 'entry' ('entry'), void )
    ParamList
    CompoundStatement
      StatementList
        StatementList
          VarDecl( 'a', Integer[] )
        Report
          BinOp( ADD )
            BinOp( ADD )
              Var('a')
              ConstString( '; size ' )
            StructMemberOp( 'size' )
              Var('a')
[]; size 0
